js/start.js

```diff
@@ -51,15 +48,55 @@ app.simpleMessageModal.remove = () => {
   throw new Error('This is a shared instance that should not be removed.');
 };
 
-const pageNav = new PageNav();
-$('#pageNavContainer').append(pageNav.render().el);
+app.pageNav = new PageNav();
+$('#pageNavContainer').append(app.pageNav.render().el);
 
-app.router = new ObRouter({
-  usersCl,
-  pageNavVw: pageNav,
-});
+app.router = new ObRouter({ usersCl });
+
+// get the server config
+$.get(app.getServerUrl('ob/config')).done((data) => {
+  app.profile = new Profile({ id: data.guid });
 
-app.loadingModal.close();
+  // todo: for now busting cache on fetch pending
+  // issue where my server is not running the latest
+  // code which solves this.
+  app.profile.fetch({ cache: false })
```

Comment:
This is mostly temporary code until we get our actual start / server connect flow into place.